Resources/Textures/Objects/Specific/Janitorial/drain.rsi/meta.json

```diff
@@ -10,6 +10,10 @@
       {
          "name":"icon",
          "directions": 4
```

Comment:
You need to update the copyright of the meta.json you touched

Reply:
They are custom-made sprites

Reply:
Then you need to add "X made by clement-or (github) for SS14" or something